dut/drivers.wake

```diff
@@ -26,6 +26,8 @@ tuple DriverImplementation =
 global def makeDriverImplementation compatibleDevices vendor deviceName driverSources includeDirs visible =
   DriverImplementation compatibleDevices vendor deviceName driverSources includeDirs visible
 
+topic driverImplementations: DriverImplementation
```

Comment:
Whoops, should have reviewed this more closely. This should have the `global` keyword.